scripts/composer/post-install.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/sh
 
+DOCUMENTROOT=web
+
+# Prepare the scaffold files if they are not already present
+if [ ! -f $DOCUMENTROOT/index.php ]
+  then
+    composer drupal-scaffold
+    mkdir -p $DOCUMENTROOT/modules
```

Comment:
Sure; any part of this process that is completely invariant, such as creating the modules and themes directories & c., could become part of the drupal-scaffold command.